_includes/header.html

```diff
@@ -61,7 +62,14 @@
         })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
         ga('create', 'UA-44483523-1', 'auto');
         ga('send', 'pageview');
+		var a = document.getElementsByTagName(‘a’);
+		for(i = 0; i < a.length; i++){
+		if (a[i].href.indexOf(location.host) == -1 && a[i].href.match(/^http:///i)){
```

Comment:
will this work with `https://`...? e.g. https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md

Reply:
That is a good call. Let's change to https.

Reply:
how about both? this we can track both `http` and `https` links...